<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img/Images/help_img.jpeg);</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="settings">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/Images/setting_backround.jpeg);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1">
          <widget class="QCheckBox" name="melodyBtn">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Z003</family>
             <pointsize>30</pointsize>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(237, 212, 0);
border-image: url(:/img/Images/transparent.png);</string>
           </property>
           <property name="text">
            <string>Melody</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="settingsBackBtn">
           <property name="font">
            <font>
             <family>Z003</family>
             <pointsize>25</pointsize>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(245, 194, 17);
border-image: url(:/img/Images/transparent.png);</string>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="help">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/Images/help_img.jpeg);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="2">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2" rowspan="2">
          <widget class="QPlainTextEdit" name="plainTextEdit_2">
           <property name="font">
            <font>
             <family>Z003</family>
             <pointsize>20</pointsize>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/img/Images/transparent.png);
color: rgb(193, 125, 17);</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string>Move instructions Water:
Up-up
Down-down
Legt-left
Right-right</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="font">
            <font>
             <family>Z003</family>
             <pointsize>20</pointsize>
             <italic>true</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/img/Images/transparent.png);
</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string>Move instructions 
Fire:
W-up
S -down
A-left
D-right</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="backBtn">
           <property name="font">
            <font>
             <family>Z003 [urw]</family>
             <pointsize>25</pointsize>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/img/Images/transparent.png);
</string>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Z003 [UKWN]</family>
             <pointsize>23</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/img/Images/transparent.png);</string>
           </property>
           <property name="text">
            <string>   Have a good luck!</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mainManu">
      <property name="styleSheet">
       <string notr="true">border-image:url(:/img/Images/bckgnd.jpeg)</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>400</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image:url(:/img/Images/transparent.png);</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="playBtn">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Z003</family>
                <pointsize>36</pointsize>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image:url(:/img/Images/transparent.png);
color:rgb(245, 194, 17)</string>
              </property>
              <property name="text">
               <string>Play</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="settingsBtn">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Z003</family>
                <pointsize>36</pointsize>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image:url(:/img/Images/transparent.png);
color:rgb(245, 194, 17)</string>
              </property>
              <property name="text">
               <string>Settings</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="highScoresBtn">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Z003</family>
                <pointsize>36</pointsize>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image:url(:/img/Images/transparent.png);
color:rgb(245, 194, 17)</string>
              </property>
              <property name="text">
               <string>Help</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="quitBtn">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Z003</family>
                <pointsize>36</pointsize>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image:url(:/img/Images/transparent.png);
color:rgb(245, 194, 17)</string>
              </property>
              <property name="text">
               <string>Quit</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="levels">
      <property name="styleSheet">
       <string notr="true">border-image:url(:/img/Images/leveli.png)</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="9" column="2">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="level1Btn">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Z003</family>
             <pointsize>27</pointsize>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image:url(:/img/Images/transparent.png);
color:rgb(245, 194, 17)</string>
           </property>
           <property name="text">
            <string>Level 1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="2">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="Score1">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/img/Images/unlocked.png);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="2">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="2">
          <widget class="QPushButton" name="level3Btn">
           <property name="font">
            <font>
             <family>Z003</family>
             <pointsize>27</pointsize>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/img/Images/transparent.png);
color: rgb(237, 212, 0);</string>
           </property>
           <property name="text">
            <string>Level 3</string>
           </property>
          </widget>
         </item>
         <item row="9" column="3">
          <widget class="QPushButton" name="backBtn1">
           <property name="font">
            <font>
             <family>Z003</family>
             <pointsize>27</pointsize>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/img/Images/transparent.png);
color: rgb(237, 212, 0);</string>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="level2Btn">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Z003</family>
             <pointsize>27</pointsize>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/img/Images/transparent.png);
color:rgb(245, 194, 17);</string>
           </property>
           <property name="text">
            <string>Level 2</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="Score2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/img/Images/locked_grey.png)</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="Score3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/img/Images/locked_grey.png)</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
